### ISSUE TITLE
Add better labels to links on the teacher homepage

### ISSUE THREAD

**Issue by jack-skerrett-oak** [oaknational/Oak-Web-Application#1561]
- Currently, the links for suggested search are confusing, I feel they need a label describing they are suggested search terms. 
- Key stage component, I think we should have the labels be "Key stage 1, key stage 2" etc, as KS1 KS2 is harder to understand. Maybe even make the labels "Browse key stage 1 lessons"
